elizabethbott/MaizExchange
# Implement login blocker and persistent login

Things that this PR addresses:

- Implements the preliminary login flow as discussed earlier this week, blocking out users until they can log in
- Saves user access tokens across sessions, allowing users to remain logged in after restarting the app
- Adds a reusable custom button component with a few different options for styling, such as `filled`, `wide`, `disabled`, etc.

To test this PR:

- Use the full login/logout flow, make sure it works as expected
- Refresh the page/reload the app at each step to make sure your login state is persisted
